guides/common/attributes.adoc

```diff
@@ -4,7 +4,7 @@
 :DocState: nightly
 // Versions used in text and code
 :ProjectVersion: 2.3
```

Comment:
```suggestion
:ProjectVersion: 2.4
```

Reply:
Yeah this was an overlook. I wanted to do this change.